src/components/Collapsible/Collapsible.tsx

```diff
@@ -34,7 +34,7 @@ interface State {
 
 const ParentCollapsibleExpandingContext = createContext(false);
 
-class Collapsible extends React.Component<CollapsibleProps, State> {
+class CollapsibleInner extends React.Component<CollapsibleProps, State> {
```

Comment:
Kill off cases where we export a default value - now things that use AppProvider are exported with a named component, which means that the component index file doesn't need to rename the default export to a named export